runtime/interpreter/value.go

```diff
@@ -14857,14 +14857,15 @@ func NewPublicKeyValue(
 		sema.PublicKeyVerifyPoPFunction: publicKeyVerifyPoPFunction,
 	}
 
-	// Validate the public key, and initialize 'isValid' field.
-
-	publicKeyValue.SetMember(
-		interpreter,
-		getLocationRange,
-		sema.PublicKeyIsValidField,
-		validatePublicKey(interpreter, getLocationRange, publicKeyValue),
-	)
+	var err error
+	err = validatePublicKey(interpreter, getLocationRange, publicKeyValue)
```

Comment:
```suggestion
	err := validatePublicKey(interpreter, getLocationRange, publicKeyValue)
```

Reply:
Oops! I did this because the Goland debugger was complaining that it didn't know what `err` was.